src/sentry/integrations/jira/integration.py

```diff
@@ -970,6 +972,23 @@ def create_issue(self, data, **kwargs):
         # Immediately fetch and return the created issue.
         return self.get_issue(issue_key)
 
+    def raise_error(self, exc: Exception, identity: Identity | None = None) -> NoReturn:
+        """
+        Overrides the base `raise_error` method to treat ApiInvalidRequestErrors
+        as configuration errors when we don't have error field handling for the
+        response.
+
+        This is because the majority of Jira errors we receive are external
+        configuration problems, like required fields missing.
+        """
+        if isinstance(exc, ApiInvalidRequestError):
+            if exc.json:
```

Comment:
Is it reasonable to do more checking of the message like checking if "is required" is in the response from Jira (like adding to `CUSTOM_ERROR_MESSAGE_MATCHERS`)? I slightly worry about overreporting halts when they should be failures

Reply:
Yeah, this seems overly permissive for now. 🤔 I can start by logging out the invalid response bodies, then creating a list to check against. The problem is, I've seen so many distinct error messages, including some that have HTML bodies, that it'll be impossible to string match against all of them.